infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -479,14 +473,11 @@ public CorfuStoreMetadata.Timestamp preprocessAndGetTail() {
                         ReplicationModel model = entry.getPayload().getModel();
                         ReplicationSubscriber subscriber = ReplicationSubscriber.newBuilder()
                                 .setClientName(clientName).setModel(model).build();
-                        // TODO (V2): currently we don't support customized client name, default logical group subscriber
-                        //  should be removed after the grpc stream for Sink session creation is created.
                         if (model.equals(ReplicationModel.LOGICAL_GROUPS)) {
                             subscriber = getDefaultLogicalGroupSubscriber();
```

Comment:
We don't need this anymore..but we are missing the part where the sink is told about a session via the leadership query, and the calculcation of streams to replicate for that session